### ISSUE TITLE
Make aruba compatible with rspec + capybara

### ISSUE THREAD

**Issue by dg-ratiodata** [cucumber/aruba#288]
`RSpec::Matchers` and `Capybara` both implement the same methods at some extend - e.g. `all`, `within`. Find a way to work around this problem:

* Looping over all items
* Ask for help from `Rspec`-guys, rspec/rspec-expectations#820

**Comment by mattwynne** [comment]
What does this have to do with aruba? Don't those projects need to work it out between themselves?

**Comment by dg-ratiodata** [comment]
Mmmh... It's a little bit more complicated. I started to move all asserts etc. to be `RSpec`-matchers. And I really really like that approach and think this is the best way to go. As we rely heavily on `rspec-expectations` users will have problems if they use capybara + aruba in their project and want to upgrade to 0.8.0. 

I think it's no a problem we can solve, but we can make it worse for users. And btw. I'm involved in `middleman` as well, and I was the person who introduced `Capybara` to their test suite and really want them to upgrade to 0.8.0. :-) Some advertising.

**Comment by mattwynne** [comment]
For Aruba users, we could avoid the problem mixing the RSpec expectations module into a bare object that we then use to do our assertions.

e.g.:

```
match = Object.new.extend(RSpec::Expectations::Matchers) # or whatever
expect(output).to match.all(blah)
```

I think the less we can pollute the users's namespace the better anyway.

**Comment by dg-ratiodata** [comment]
Good idea, but that would be a breaking change. And it fails for the `be_`-matcher.

**Comment by dg-ratiodata** [comment]
Fixed with 0.9.0 or 0.10.0
